### PR TITLE
Add basic configs for prlint

### PR DESCRIPTION
**Pull request by DalerAsrorov**
## Types of changes

- [ ] Refactor (changes the way we code something without changing its functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] Latest master code has been merged into this branch
- [x] No commented out code (if required, place // TODO above with explanation)
- [x] No linting issues
- [x] Build is successful
- [ ] Code follows the [API Guidelines](http://britecharts.github.io/britecharts/topics-index.html#toc5__anchor)
- [x] Updated the documentation
- [x] Added tests to cover changes
- [x] All new and existing tests passed

### CODE REVIEW

File: .github/prlint.json [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    ],
+    "assignee.login": [
+        {
+            "pattern": ".+",
+            "message": "You need to assign someone"
+        }
+    ],
+    "labels.0.name": [
+        {
+            "pattern": "bug|enhancement|question|feature",
```
Comment (Golodhros) [inline .github/prlint.json:28]: Let's add a couple more here:
build | dependencies | design | docs

and change "feature" into "proposal"

File: .github/prlint.json [hunk 2]
```diff
@@ -0,0 +1,32 @@
+{
+    "title": [
+        {
+            "pattern": "^(docs|feat|fix|dev|test):\\s",
```
Comment (Golodhros) [inline .github/prlint.json:4]: Let's add 'ref', 'build' and 'perf' too.

File: .github/prlint.json [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+    ],
+    "body": [
+        {
+            "pattern": ".{1,}",
+            "message": "You need to have some content in the description."
+        }
+    ],
+    "head.ref": [
+        {
+            "pattern": "^(docs|feat|fix|dev|test)/",
```
Comment (Golodhros) [inline .github/prlint.json:16]: Let's add 'ref', 'build' and 'perf' too.

File: .github/prlint.json [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+            "message": "Your branch name is invalid. Please prefix it with docs|feat|fix|dev|test."
+        }
+    ],
+    "assignee.login": [
+        {
+            "pattern": ".+",
+            "message": "You need to assign someone"
```
Comment (Golodhros) [inline .github/prlint.json:23]: Not sure about this one. 
I guess for PRs they would assign themselves?
I guess is not necessary! Thoughts?

File: .github/prlint.json (hunk 2 above)
Comment (Golodhros) [reply .github/prlint.json:4]: Disregard that comment!

I was thinking on simplifying it a bit and make these map to the types of issues on our labels (https://github.com/britecharts/britecharts/labels)

So, that would become:
fix - for bug fixes
feat - for features
docs - for documentation stuff
main - for all the rest (refactors, build, tests, dev environment, etc.)

What do you think?

File: .github/prlint.json (hunk 3 above)
Comment (Golodhros) [inline .github/prlint.json:16]: Same as on the title prefixes

**Comment by Golodhros** [comment]
I just added my thoughts on https://docs.google.com/document/d/1NdMMoTcgvKN4S7egVruXmrGdJprbJ1kwmTqu8646LCU/edit?usp=sharing

Please take a look and comment!

File: .github/prlint.json (hunk 4 above)
Comment (DalerAsrorov) [reply .github/prlint.json:23]: I think it would be great if they assign themselves so that we know who those commits belong to.

Comment (Golodhros) [reply .github/prlint.json:23]: But we know it already, no?
We have that info in the commits themselves, we won't do anything with this information.

**Comment by Golodhros** [comment]
The PRLint app stopped working lately, proposing another option on https://github.com/britecharts/britecharts/pull/835

**Pull request closed without merging**